featuretools/utils/cli_utils.py

```diff
@@ -0,0 +1,42 @@
+import locale
+import os
+import platform
+import struct
+import sys
+
+import pkg_resources
+
+
+# Modified from here
+# https://github.com/pandas-dev/pandas/blob/d9a037ec4ad0aab0f5bf2ad18a30554c38299e57/pandas/util/_print_versions.py#L11
+def get_sys_info():
```

Comment:
can we add a test case that calls `get_installed_packages` and `get_sys_info` even if it doesn’t check the outputs? that would at least catch any runtime errors

Reply:
Fixed.